iXBRLViewerPlugin/viewer/src/js/search.js

```diff
@@ -38,6 +39,12 @@ ReportSearch.prototype.buildSearchIndex = function () {
         doc.label = l;
         doc.ref = f.concept().referenceValuesAsString();
         docs.push(doc);
+
+        var p = f.period();
+        if (p) {
+            this.periods[p.key()] = p.toString();
```

Comment:
Where is this used?

Reply:
It's used to populate the period drop down here:

https://github.com/Workiva/ixbrl-viewer/pull/108/files#diff-78e5e99f173b31835d4b2e2dc5defcbcR217